qiskit/transpiler/preset_passmanagers/__init__.py

```diff
@@ -148,6 +149,10 @@ def generate_preset_pass_manager(
             the ``unitary_synthesis`` argument. As this is custom for each
             unitary synthesis plugin refer to the plugin documentation for how
             to use this option.
+        hls_config (HLSConfig): An optional configuration class that will be passed directly
+            to the high-level synthesis transformation pass. This configuration
+            class allows to specify for various high-level objects the lists of
+            synthesis algorithms and their parameters.
```

Comment:
Can you also add a link to the `HighLevelSynthesis` pass here?

Reply:
done